README/ReleaseNotes/v632/index.md

```diff
@@ -71,6 +71,19 @@ TChain::Add("/path/to/tree/subdir[0-9]/*.root")
 ```
 This grabs all the root files in subdirectories that have a name starting with `subdir` and ending with some digit.
 
+## RNTuple
+ROOT's experimental successor of TTree has seen a number of updates since the last release. Specifically, 6.32 includes the following changes:
+
+- A major refactoring of the interface, improving consistency across different parts and improving overall robustness. **Note that this is a breaking change with regard to 6.30!**
+- Support has been added for several new field types: `std::unordered_set<T>`, `std::map<K,V>`, `std::unordered_map<K,V>`
+- A new `RNTupleInspector` utility class has been added, to provide information about the on-disk metadata of an RNTuple.
+- A new `RNTupleParallelWriter` class has been added, providing (initial) support for parallel writing of RNTuples.
+- A new static method `RFieldBase::Check()` has been added, which produces a support status report of a type with regards to RNTuple I/O.
+- A new internal `RNTupleMerger` class has been added, enabling the merging of different page sources into one page sink.
+- Many bug fixes and improvements.
```

Comment:
Small addition:
- It is possible to use the `RNTupleView` with an external address with type erasure, e.g. `std::shared_ptr<void> data{new float()}; auto view = reader->GetView("pt", data);`. This enables use cases such as reading one specific entry of one specific field into a previously allocated memory location.